### PR TITLE
Add walkthrough guide

### PR DESCRIPTION
**Pull request by dannynotsmart**
This PR adds a walkthrough of all the bot's commands, in the `walkthrough` folder. It also adds `device_list.md` to the guide

### CODE REVIEW

File: guides/device_list.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+**Here is a list of all the devices you can register when using the `/set` command:**
```
Comment (PythonCoderAS) [inline guides/device_list.md:1]: Does this need to be bolded?
```suggestion
Here is a list of all the devices you can register when using the `/set` command:
```

File: guides/walkthrough/page_1.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+**This is the complete walkthrough of the commands of the HealthScreening Bot.**
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:1]:
```suggestion
**This is the complete walkthrough of the commands of the Health Screening Bot.**
```

File: guides/walkthrough/page_1.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+This bot is fairly simple and self-explanatory, but this guide serves as an complete walkthrough of the commands.
+`Highlighted text` is a command name
+- dashed text are command arguments!
+
+
+**Automated Health Screening Commands:**
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:7]:
```suggestion
__**Automated Health Screening Commands**__:
```

File: guides/walkthrough/page_1.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+`set_auto` : Sets the information used for the automated health screening
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:8]:
```suggestion
`/set_auto`: Sets the information used for the automated health screening.
```

File: guides/walkthrough/page_1.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+Paremeters:
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:9]:
```suggestion
__Parameters__:
```

Comment (PythonCoderAS) [reply guides/walkthrough/page_1.md:9]: There was a typo, so I took the liberty of fixing it.

File: guides/walkthrough/page_1.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+- first_name : Your first name
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:10]:
```suggestion
- **first_name**: Your first name.
```

File: guides/walkthrough/page_1.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+- last_name : Your last name
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:11]:
```suggestion
- **last_name**: Your last name.
```

File: guides/walkthrough/page_1.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+- email : Your email
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:12]:
```suggestion
- **email**: Your email. (If you do not want to get emails from the bot, just set it to a fake email.)
```

File: guides/walkthrough/page_1.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+- dashed text are command arguments!
+
+
+**Automated Health Screening Commands:**
+`set_auto` : Sets the information used for the automated health screening
+Paremeters:
+- first_name : Your first name
+- last_name : Your last name
+- email : Your email 
+- vaccinated : Whether or not you are vaccinated. This should be either __True__ or __False__
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:13]:
```suggestion
- **vaccinated**: Whether or not you are vaccinated. This should be either __True__ or __False__.
```

File: guides/walkthrough/page_1.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
+**This is the complete walkthrough of the commands of the HealthScreening Bot.**
+This bot is fairly simple and self-explanatory, but this guide serves as an complete walkthrough of the commands.
+`Highlighted text` is a command name
+- dashed text are command arguments!
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:4]:
```suggestion
- dashed text are command arguments.
```

File: guides/walkthrough/page_1.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+
+
+**Automated Health Screening Commands:**
+`set_auto` : Sets the information used for the automated health screening
+Paremeters:
+- first_name : Your first name
+- last_name : Your last name
+- email : Your email 
+- vaccinated : Whether or not you are vaccinated. This should be either __True__ or __False__
+- ephemeral : Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:14]:
```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

File: guides/walkthrough/page_1.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:15]:
```suggestion

```

File: guides/walkthrough/page_1.md [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+
+`generate_auto` : Generate an health screening using your information set with `set_auto`.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:17]:
```suggestion
`/generate_auto`: Generate a health screening using the information set with `/set_auto`.
```

File: guides/walkthrough/page_1.md [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+Paremeters:
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:18]:
```suggestion
__Paremeters__:
```

File: guides/walkthrough/page_1.md [hunk 15]
```diff
@@ -0,0 +1,19 @@
...
+- ephemeral : Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:19]:
```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

File: guides/walkthrough/page_2.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
+`set` : Sets additional information for the automated health screening.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:1]:
```suggestion
__**Other Useful Screening-Related Commands**__:
`/set`: Sets additional information for the automated health screening.
```

File: guides/walkthrough/page_2.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+Paremeters (All optional by the way):
+- device: The name of the device for the health screening to "use". To view the full device list, do `/guide name:device_list`.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:3]:
```suggestion
- **device**: The name of the device for the health screening to "use". To view the full device list, do `/guide name:device_list`. *Note*: The device is used with `/generate_once` as well, so deleting your auto information will not completely reset your device. The default device used is `iPhone 11`.
```

File: guides/walkthrough/page_2.md [hunk 18]
```diff
@@ -0,0 +1,20 @@
...
+- hour : The hour to run the health screening at. Number should be in between 0 - 23.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:4]:
```suggestion
- **hour**: The hour to run the health screening at. Should be a number in the range of 0-23. *Note*: Changing the hour does not set the minute to zero. The default hour is 5.
```

File: guides/walkthrough/page_2.md [hunk 19]
```diff
@@ -0,0 +1,20 @@
...
+- minute : The minute to run the health screening at. Number  should be in between 0 - 59.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:5]:
```suggestion
- **minute**: The minute to run the health screening at. Should be a number between 0 - 59.
```

File: guides/walkthrough/page_2.md [hunk 20]
```diff
@@ -0,0 +1,20 @@
...
+*Note*: Together, the hour and time should represent the time you get your health screening at, such as 5:20, or 23:35 (or 11:35 PM)
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:6]:
```suggestion
Together, the hour and minute should represent the time you get your health screening at, such as 5:20, or 23:35 (11:35 PM)
```

File: guides/walkthrough/page_2.md [hunk 21]
```diff
@@ -0,0 +1,20 @@
...
+- type : The type of screening to generate. Available options are __guest__, __student__, or __employee__, but defaults to __guest__.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:7]:
```suggestion
- **type**: The type of screening to generate. Available options are __guest__, __student__, or __employee__. The default is __guest__.
```

File: guides/walkthrough/page_2.md [hunk 22]
```diff
@@ -0,0 +1,20 @@
...
+- email_only : Whether the health screening should be emailed only or not. If this is True, the bot will **not** DM you the health screening on Discord. This should be either __True__ or __False__, but defaults to False.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:8]:
```suggestion
- **email_only**: Whether the health screening should be only emailed or emailed and screenshotted. If this is __True__, the bot will **not** automatically DM you the health screening on Discord. This should be either __True__ or __False__. The default is __False__. The `/generate_auto` command will produce a screenshot regardless of what this is set to.
```

File: guides/walkthrough/page_2.md [hunk 23]
```diff
@@ -0,0 +1,20 @@
...
+- paused : Whether to pause the auto health screenings. If this is True, you will **not** receive any automated screenings. This should be either __True__ or __False__, but defaults to False.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:9]:
```suggestion
- **paused**: Whether to pause the auto health screenings. If this is True, you will **not** receive any automated screenings, including emails. This should be either __True__ or __False__. The default is __False__.
```

File: guides/walkthrough/page_2.md [hunk 24]
```diff
@@ -0,0 +1,20 @@
...
+*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If False, you will **not** receive any automated screenings on that day.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:10]:
```suggestion
*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If __False__, you will **not** receive any automated screenings on that day.
```

Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:10]:
```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If False, you will **not** receive any automated screenings on that day.
```

File: guides/walkthrough/page_2.md [hunk 25]
```diff
@@ -0,0 +1,20 @@
...
+- sunday - monday - tuesday - wednesday - thursday - friday - saturday
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:11]:
```suggestion
- **sunday**: Defaults to __False__.
- **monday**: Defaults to __True__.
- **tuesday**: Defaults to __True__.
- **wednesday**: Defaults to __True__.
- **thursday**: Defaults to __True__.
- **friday**: Defaults to __True__.
- **saturday**: Defaults to __False__.
```

File: guides/walkthrough/page_2.md [hunk 26]
```diff
@@ -0,0 +1,20 @@
...
+
+
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:13]:
```suggestion

```

File: guides/walkthrough/page_2.md [hunk 27]
```diff
@@ -0,0 +1,20 @@
...
+`generate_once` : Generate a singular health screening without using your information set with `set_auto`.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:14]:
```suggestion
`/generate_once`: Generate a singular health screening without using your information set with `/set_auto`. If set, the device **will** be considered.
```

File: guides/walkthrough/page_2.md [hunk 28]
```diff
@@ -0,0 +1,20 @@
...
+Parameters:
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:15]:
```suggestion
__Parameters__:
```

File: guides/walkthrough/page_2.md [hunk 29]
```diff
@@ -0,0 +1,20 @@
...
+- first_name : Your first name
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:16]:
```suggestion
- **first_name**: Your first name.
```

File: guides/walkthrough/page_2.md [hunk 30]
```diff
@@ -0,0 +1,20 @@
...
+- last_name : Your last name
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:17]:
```suggestion
- **last_name**: Your last name.
```

File: guides/walkthrough/page_2.md [hunk 31]
```diff
@@ -0,0 +1,20 @@
...
+- email : Your email
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:18]:
```suggestion
- **email**: Your email.
```

File: guides/walkthrough/page_2.md [hunk 32]
```diff
@@ -0,0 +1,20 @@
...
+*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If False, you will **not** receive any automated screenings on that day.
+- sunday - monday - tuesday - wednesday - thursday - friday - saturday
+
+
+`generate_once` : Generate a singular health screening without using your information set with `set_auto`.
+Parameters:
+- first_name : Your first name
+- last_name : Your last name
+- email : Your email 
+- vaccinated : Whether or not you are vaccinated. This should be either __True__ or __False__
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:19]:
```suggestion
- **vaccinated**: Whether or not you are vaccinated. This should be either __True__ or __False__.
```

File: guides/walkthrough/page_1.md [hunk 33]
```diff
@@ -0,0 +1,19 @@
+**This is the complete walkthrough of the commands of the HealthScreening Bot.**
+This bot is fairly simple and self-explanatory, but this guide serves as an complete walkthrough of the commands.
+`Highlighted text` is a command name
+- dashed text are command arguments!
+
+
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:6]:
```suggestion

```

File: guides/walkthrough/page_2.md [hunk 34]
```diff
@@ -0,0 +1,20 @@
...
+- ephemeral : Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:20]:
```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

File: guides/walkthrough/page_3.md [hunk 35]
```diff
@@ -0,0 +1,12 @@
+**Miscellaneous**
+*Note* All commands in this category have an ephemeral argument, which hides the content sent from everyone else if True. This argument will not be listed in these parameters.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:2]: This isn't true for stats.
```suggestion
```

File: guides/walkthrough/page_3.md [hunk 36]
```diff
@@ -0,0 +1,12 @@
+**Miscellaneous**
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:1]:
```suggestion
__**Miscellaneous Commands**__:
```

File: guides/walkthrough/page_3.md [hunk 37]
```diff
@@ -0,0 +1,12 @@
...
+*Note* All commands in this category have an ephemeral argument, which hides the content sent from everyone else if True. This argument will not be listed in these parameters.
+`guide`: Sends guides from the bot's github.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:3]:
```suggestion
`/guide`: Sends a guide from a list of approved guides. Guides can be contributed by anyone to the bot's GitHub repository.
```

File: guides/walkthrough/page_3.md [hunk 38]
```diff
@@ -0,0 +1,12 @@
...
+Paremeters: 
+- name : Name of the guide to send
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:5]:
```suggestion
- **name**: The name of the guide to send.
```

File: guides/walkthrough/page_3.md [hunk 39]
```diff
@@ -0,0 +1,12 @@
...
+- paginate : Whether to paginate the guide. This is an optional argument.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:6]:
```suggestion
- **paginate**: Whether to paginate the guide. This is an optional argument and should be either __True__ or __False__. The default is __True__. Note: One-page guides will not have any paginator buttons regardless of the value provided to this argument.
```

File: guides/walkthrough/page_3.md [hunk 40]
```diff
@@ -0,0 +1,12 @@
...
+
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:7]:
```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

File: guides/walkthrough/page_3.md [hunk 41]
```diff
@@ -0,0 +1,12 @@
...
+
+`profile` : Sends the information set with the `set` command.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:9]:
```suggestion
`/profile`: Shows an overview of all information stored for the user, including information submitted via `/set_auto` and `/set`.
```

File: guides/walkthrough/page_3.md [hunk 42]
```diff
@@ -0,0 +1,12 @@
...
+
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:10]:
```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

File: guides/walkthrough/page_3.md [hunk 43]
```diff
@@ -0,0 +1,12 @@
...
+
+`stats` : Send bot stats. This does not have the ephemeral argument (why would it?)
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:12]:
```suggestion
`/stats` : Send bot stats. This does not have the ephemeral argument (why would it?)
```

**Review by PythonCoderAS — CHANGES_REQUESTED** [review]
Thanks for submitting the guide. I would like to mention that there is a `/delete_auto` command, which deletes all stored information from `/set_auto` and `/set` excluding the device (and is not ephemeral). Most of the suggested changes are minor formatting changes like bolding parameters and underlining the Parameters sections.

File: guides/walkthrough/page_1.md (hunk 5 above)
Comment (dannynotsmart) [reply guides/walkthrough/page_1.md:7]: Thank you

**Review by dannynotsmart — COMMENTED** [review]
How in the world did I open a review

File: guides/walkthrough/page_1.md [hunk 44]
```diff
@@ -0,0 +1,16 @@
+**This is the complete walkthrough of the commands of the Health Screening Bot.**
+This bot is fairly simple and self-explanatory, but this guide serves as an complete walkthrough of the commands.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:2]:
```suggestion

This bot is fairly simple and self-explanatory, but this guide serves as an complete walkthrough of the commands.
```

File: guides/walkthrough/page_1.md [hunk 45]
```diff
@@ -0,0 +1,16 @@
...
+`Highlighted text` is a command name
+- dashed text are command arguments.
+__**Automated Health Screening Commands**__:
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:5]:
```suggestion

__**Automated Health Screening Commands**__:
```

File: guides/walkthrough/page_1.md [hunk 46]
```diff
@@ -0,0 +1,16 @@
...
+`/set_auto`: Sets the information used for the automated health screening.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:6]:
```suggestion

`/set_auto`: Sets the information used for the automated health screening.
```

File: guides/walkthrough/page_1.md [hunk 47]
```diff
@@ -0,0 +1,16 @@
...
+__Parameters__:
+- **first_name**: Your first name.
+- **last_name**: Your last name.
+- **email**: Your email. (If you do not want to get emails from the bot, just set it to a fake email.)
+- **vaccinated**: Whether or not you are vaccinated. This should be either __True__ or __False__.
+- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
+
+`/generate_auto`: Generate a health screening using the information set with `/set_auto`.
+__Paremeters__:
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:15]:
```suggestion
__Parameters__:
```

File: guides/walkthrough/page_2.md [hunk 48]
```diff
@@ -0,0 +1,25 @@
+__**Other Useful Screening-Related Commands**__:
+`/set`: Sets additional information for the automated health screening.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:2]:
```suggestion

`/set`: Sets additional information for the automated health screening.
```

File: guides/walkthrough/page_2.md [hunk 49]
```diff
@@ -0,0 +1,25 @@
...
+Paremeters (All optional by the way):
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:3]:
```suggestion
__Parameters__ (All optional by the way):
```

File: guides/walkthrough/page_2.md [hunk 50]
```diff
@@ -0,0 +1,25 @@
...
+- **paused**: Whether to pause the auto health screenings. If this is True, you will **not** receive any automated screenings, including emails. This should be either __True__ or __False__. The default is __False__.
+*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If __False__, you will **not** receive any automated screenings on that day.
+- **sunday**: Defaults to __False__.
+- **monday**: Defaults to __True__.
+- **tuesday**: Defaults to __True__.
+- **wednesday**: Defaults to __True__.
+- **thursday**: Defaults to __True__.
+- **friday**: Defaults to __True__.
+- **saturday**: Defaults to __False__.
+`/generate_once`: Generate a singular health screening without using your information set with `/set_auto`. If set, the device **will** be considered.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:19]:
```suggestion

`/generate_once`: Generate a singular health screening without using your information set with `/set_auto`. If set, the device **will** be considered.
```

File: guides/walkthrough/page_3.md [hunk 51]
```diff
@@ -0,0 +1,11 @@
+__**Miscellaneous Commands**__:
+`/guide`: Sends a guide from a list of approved guides. Guides can be contributed by anyone to the bot's GitHub repository.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:2]:
```suggestion

`/guide`: Sends a guide from a list of approved guides. Guides can be contributed by anyone to the bot's GitHub repository.
```

File: guides/walkthrough/page_3.md [hunk 52]
```diff
@@ -0,0 +1,11 @@
...
+Paremeters:
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:3]:
```suggestion
__Parameters__:
```

File: guides/walkthrough/page_3.md [hunk 53]
```diff
@@ -0,0 +1,11 @@
...
+`/guide`: Sends a guide from a list of approved guides. Guides can be contributed by anyone to the bot's GitHub repository.
+Paremeters: 
+- **name**: The name of the guide to send.
+- **paginate**: Whether to paginate the guide. This is an optional argument and should be either __True__ or __False__. The default is __True__. Note: One-page guides will not have any paginator buttons regardless of the value provided to this argument.
+- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
+
+`/profile`: Shows an overview of all information stored for the user, including information submitted via `/set_auto` and `/set`.
+- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
+
+`/stats` : Send bot stats. This does not have the ephemeral argument (why would it?)
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:11]:
```suggestion
`/stats`: Send bot stats. This does not have the ephemeral argument (why would it?)
```

File: guides/walkthrough/page_2.md [hunk 54]
```diff
@@ -0,0 +1,25 @@
...
+- **minute**: The minute to run the health screening at. Should be a number between 0 - 59.
+Together, the hour and minute should represent the time you get your health screening at, such as 5:20, or 23:35 (11:35 PM)
+- **type**: The type of screening to generate. Available options are __guest__, __student__, or __employee__. The default is __guest__.
+- **email_only**: Whether the health screening should be only emailed or emailed and screenshotted. If this is __True__, the bot will **not** automatically DM you the health screening on Discord. This should be either __True__ or __False__. The default is __False__. The `/generate_auto` command will produce a screenshot regardless of what this is set to.
+- **paused**: Whether to pause the auto health screenings. If this is True, you will **not** receive any automated screenings, including emails. This should be either __True__ or __False__. The default is __False__.
+*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If __False__, you will **not** receive any automated screenings on that day.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_2.md:11]:
```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If False, you will **not** receive any automated screenings on that day.
```

File: guides/walkthrough/page_3.md [hunk 55]
```diff
@@ -0,0 +1,11 @@
+__**Miscellaneous Commands**__:
+`/guide`: Sends a guide from a list of approved guides. Guides can be contributed by anyone to the bot's GitHub repository.
+Paremeters: 
+- **name**: The name of the guide to send.
+- **paginate**: Whether to paginate the guide. This is an optional argument and should be either __True__ or __False__. The default is __True__. Note: One-page guides will not have any paginator buttons regardless of the value provided to this argument.
+- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
+
+`/profile`: Shows an overview of all information stored for the user, including information submitted via `/set_auto` and `/set`.
+- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_3.md:9]:
```suggestion
__Parameters__:
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

**Review by PythonCoderAS — CHANGES_REQUESTED** [review]

**Comment by PythonCoderAS** [comment]
Can you add in `/delete_auto` as well?

**Comment by dannynotsmart** [comment]
> Can you add in `/delete_auto` as well?

Done.

File: guides/walkthrough/page_1.md [hunk 56]
```diff
@@ -0,0 +1,19 @@
+**This is the complete walkthrough of the commands of the Health Screening Bot.**
+
+This bot is fairly simple and self-explanatory, but this guide serves as an complete walkthrough of the commands.
+`Highlighted text` is a command name
```
Comment (PythonCoderAS) [inline guides/walkthrough/page_1.md:4]:
```suggestion
`Highlighted text` is a command name.
```

**Review by PythonCoderAS — APPROVED** [review]
Great! Just one final fix and then we can get ready to merge.

**Pull request merged**